batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/dao/EmployeeDao.java

```diff
@@ -0,0 +1,143 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.constants.ModelConstants;
+import com.lftechnology.batch7crud.db.DBConnection;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Employee;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by Grishma Shrestha grishmashrestha@lftechnology.com on 1/18/16.
+ */
+
+public class EmployeeDao {
+  private static final Logger LOGGER = Logger.getLogger(Employee.class.getName());
```

Comment:
Should be EmployeeDao instead of Employee.class.getName(). Copy paste effect :smile: